cfme/web_ui/menu.py

```diff
@@ -22,7 +22,8 @@ def get_current_toplevel_name():
 def _tree_func_with_grid(*args):
     def f():
         accordion.tree(*args)
-        toolbar.select('Grid View')
+        if toolbar.exists('Grid View'):
+            toolbar.select('Grid View')
```

Comment:
If the button doens't exist, that's a bug, but if it's greyed out, that's a different matter. Unless......have they started removing the button if it is the one that is selected?